analysis/contribution_bounders.py

```diff
@@ -16,24 +16,24 @@
 import numpy as np
 from pipeline_dp import contribution_bounders
 from pipeline_dp import sampling_utils
-from typing import Iterable
+from typing import Iterable, Union
 
 
-class AnalysisContributionBounder(contribution_bounders.ContributionBounder):
+class L0LinfAnalysisContributionBounder(
+        contribution_bounders.ContributionBounder):
     """'Bounds' the contribution by privacy_id per and cross partitions.
 
     Because this is for Utility Analysis, it doesn't actually ensure that
-    contribution bounds are enforced. Instead, it keeps track of probabilities
-    different data points are kept under per-partition (Linf) and
-    cross-partition (L0) contribution bounding.
+    contribution bounds are enforced. Instead, it keeps track the information
+    needed for computing impact of contribution bounding.
 
     If partitions_sampling_prob < 1.0, partitions are subsampled. This sampling
     is deterministic and depends on partition key.
-
-    Only works for count at the moment.
     """
 
-    def __init__(self, partitions_sampling_prob: float):
+    def __init__(self,
+                 partitions_sampling_prob: float,
+                 perform_cross_partition_contribution_bounding: bool = True):
```

Comment:
why do we need this param if we have `LinfAnalysisContributionBounder`? If it is false then it is the smae as `LinfAnalysisContributionBounder`.